about.md

```diff
@@ -12,9 +12,9 @@ scribble: scribble6
 
 So what is Rails Girls Summer of Code all about?
 
-Rails Girls Summer of Code is a global fellowship program aimed at bringing
+Rails Girls Summer of Code is an [award-winning](https://rubyheroes.com/heroes/2016){:target="_blank"} global fellowship program aimed at bringing
```

Comment:
I had completely forgotten about the blog post actually (originally I had linked to the press release, but a link to a dropbox pdf looked weird). I will change this and link to the blog post!